pkg/deploy/lattice/listener_manager.go

```diff
@@ -99,53 +109,46 @@ func (d *defaultListenerManager) Create(ctx context.Context, listener *latticemo
 	}
 
 	resp, err := d.cloud.Lattice().CreateListener(&listenerInput)
+	if err != nil {
+		d.log.Errorf("Error during creation of Listener %s-%s", listenerSpec.Name, listenerSpec.Namespace)
```

Comment:
I would return error here

Reply:
Would like to avoid behavioural changes in this PR, just improving logging/readability of code.

Reply:
in general for refactoring you are right, but in this particular example resp is used as non-nil value few lines below, it will panic on nil.

Reply:
Fair point, fixed next PR!